### PR TITLE
Add new car

### PR DESCRIPTION
**Pull request by maksymvasylyev**

### CODE REVIEW

File: components/AddCarForm/AddCarList.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const StyledAddCarList = styled.ul`
+  list-style: none;
+  padding: 0px;
+  gap: 10px;
+  display: grid;
+  font-size: 1em;
+  justify-items: center;
+  grid-template-columns: 1fr;
+  border: 3px, solid, red;
```
Comment (djfarly) [inline components/AddCarForm/AddCarList.js:17]: I don't think this works with `,`

```suggestion
  border: 3px solid red;
```

File: components/Compare/Compare.js [hunk 2]
```diff
@@ -25,6 +25,15 @@ const StyledNameCompareH4 = styled.h4`
...
 
+const StyledYourCarDiv = styled.div`
+  /* background-color: transparent;
+  border: none;
+  display: flex;
+  position: relative; */
+  font-size: 1.5em;
+  margin-bottom: 13px;
+`;
+
```
Comment (djfarly) [inline components/Compare/Compare.js:36]: try not to merge commented out code:

```suggestion
const StyledYourCarDiv = styled.div`
  font-size: 1.5em;
  margin-bottom: 13px;
`;

```

**Review by djfarly — APPROVED** [review]
Hej Max,

your code looks good! I have some small notes but I'll approve the PR anyways :)

File: components/Compare/Compare.js (hunk 2 above)
Comment (maksymvasylyev) [reply components/Compare/Compare.js:36]: Thank you very much!!!

**Pull request merged**